frontend/src/js/controllers/changePwdCtrl.js

```diff
@@ -7,67 +7,73 @@
         .module('evalai')
         .controller('ChangePwdCtrl', ChangePwdCtrl);
 
-    ChangePwdCtrl.$inject = ['utilities', 'validationSvc', '$state', '$http', '$rootScope'];
+    ChangePwdCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
 
-    function ChangePwdCtrl(utilities, validationSvc, $state, $http, $rootScope) {
+    function ChangePwdCtrl(utilities, $state, $http, $rootScope) {
         var vm = this;
         var userKey = utilities.getData('userKey');
         vm.wrnMsg = {};
         vm.isValid = {};
-
+        vm.user = {};
         // function to change password
-        vm.changePassword = function() {
+        vm.changePassword = function(resetconfirmFormValid) {
+          if(resetconfirmFormValid){
             vm.loginContainer = angular.element('.change-passowrd-card');
-
+            vm.startLoader("Changing Your Password");
```

Comment:
`vm.startLoader` is not defined anywhere in this controller. that's why nothing happen when you submit form. 
See `AuthCtrl`. you can find this function there. Copy and paste in this Controller.